Subject: Move default rpmdb path to user home
From: Michal Čihař <nijel@debian.org>

We don't want to use system database, because we don't want users to install
rpm packages on Debian.

--- a/macros.in
+++ b/macros.in
@@ -161,7 +161,7 @@
 %_bzip2bin		%{__bzip2}
 
 #	The location of the rpm database file(s).
-%_dbpath		%{_var}/lib/rpm
+%_dbpath		%(echo $HOME/.rpmdb)
 
 #	The location of the rpm database file(s) after "rpm --rebuilddb".
 %_dbpath_rebuild	%{_dbpath}
